components/UpsellPlusModal.vue

```diff
@@ -95,6 +95,7 @@
         class="w-full mt-4 laptop:mt-6"
         :is-monthly-hidden="!shouldShowMonthlyPlan"
         :is-yearly-hidden="!shouldShowYearlyPlan"
+        :is-upsell="true"
```

Comment:
if gift nftClassId is not set we still want to show $1/30d for yearly, can we expose more control here to do that?